apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-checkbox-example.component.html

```diff
@@ -0,0 +1,301 @@
+        <fdp-checkbox
+            [id]="'checkbox-7'"
+            [name]="'checkbox-7'"
+            [label]="'Checkbox7'"
+            [value]="'Checkbox7'"
+            [(ngModel)]="example6"
+        ></fdp-checkbox>
+    </fieldset>
+
+    <fieldset fd-fieldset>
+        <span>Example 7.1: Default formcontrol value given for Example 7.1 and Example 7.2</span>
+        <fdp-checkbox
+            [id]="'checkbox-8'"
+            [name]="'checkbox-8'"
+            [label]="'Checkbox8'"
+            [value]="'Checkbox8'"
+            [(ngModel)]="example7"
+        ></fdp-checkbox>
+    </fieldset>
+
+    <fieldset fd-fieldset>
+        <span>Example 7.2</span>
+        <fdp-checkbox
+            [id]="'checkbox-9'"
+            [name]="'checkbox-9'"
+            [label]="'Checkbox9'"
+            [value]="'Checkbox9'"
+            [(ngModel)]="example7"
+        ></fdp-checkbox>
+    </fieldset>
+
+    <fieldset fd-fieldset>
+        <span>Example 7.3</span>
+        <fdp-checkbox
+            [id]="'checkbox-10'"
+            [name]="'checkbox-10'"
+            [label]="'Checkbox10'"
+            [value]="'Checkbox10'"
+            [(ngModel)]="example7"
+        ></fdp-checkbox>
+    </fieldset>
+
+    <fieldset fd-fieldset>
+        <span>Example for Disabled Checkbox</span>
+        <fdp-checkbox
+            [id]="'checkbox-010'"
+            [name]="'checkbox-010'"
+            [label]="'Checkbox010'"
+            [value]="'Checkbox010'"
+            [disabled]="true"
+        ></fdp-checkbox>
+    </fieldset>
+</form>
+<div>Example 1: {{ example1 }}</div>
+<div>Example 2: {{ example2 }}</div>
+<div>Example 3: {{ example3 }}</div>
+<div>Example 4: {{ example4 }}</div>
+<div>Example 5: {{ example5 }}</div>
+<div>Example 6: {{ example6 }}</div>
+<div>Example 7: {{ example7 }}</div>
+
+<br />
+<br />
+
+<h2>Compact Checkbox with Reactive form</h2>
+<form [formGroup]="customForm">
+    <fieldset fd-fieldset>
+        <span>Example 8: Binary Checkbox with default false value</span>
+        <fdp-checkbox
+            [id]="'checkbox-11'"
+            [name]="'checkbox-11'"
+            [label]="'Checkbox11'"
+            [isBinary]="true"
+            [contentDensity]="'compact'"
+            formControlName="example8"
+        ></fdp-checkbox>
+    </fieldset>
+
+    <fieldset fd-fieldset>
+        <span>Example 9: Binary Checkbox with default true value</span>
+        <fdp-checkbox
+            [id]="'checkbox-12'"
+            [name]="'checkbox-12'"
+            [label]="'Checkbox12'"
+            [isBinary]="true"
+            [contentDensity]="'compact'"
+            formControlName="example9"
+        ></fdp-checkbox>
+    </fieldset>
+
+    <fieldset fd-fieldset>
+        <span>Example 10: Binary Checkbox without default value</span>
+        <fdp-checkbox
+            [id]="'checkbox-13'"
+            [name]="'checkbox-13'"
+            [label]="'Checkbox13'"
+            [isBinary]="true"
+            [contentDensity]="'compact'"
```

Comment:
this should have the same approach as disabled and all other compacts with a boolean

Reply:
In Platform library implementation, we have decided to have contentDensity. as in future there may be more options than "compact" and "cozy".